k-distribution/include/builtin/kast.k

```diff
@@ -63,32 +63,39 @@ module KSEQ
 endmodule
 
 
-// To be used when parsing/pretty-printing symbolic configurations
-module KSEQ-SYMBOLIC
-  imports KSEQ
+module ML-SYNTAX
+  imports BASIC-K
+
   syntax #KVariable ::= r"(?<![A-Za-z0-9_\\$!\\?])(\\!|\\?)?([A-Z][A-Za-z0-9'_]*|_)"   [token, prec(1), autoReject]
```

Comment:
I don't want this production in this module because #KVariable is not a real sort.